proposals/acp/wac-acp-diff-story.md

```diff
@@ -0,0 +1,134 @@
+# ACP Compared to Web Access Control
+
+This document is intended to reflect the following authoritative-ish definitions:
+- [W3C wiki](http://www.w3.org/wiki/WebAccessControl)
+- [solid/specification](http://github.com/solid/specification)
+- [sold/web-access-control](http://github.com/solid/solid-spec)
+The goal is to ground conversations which compare WAC and ACP.
+This is a work in progress.
+
+Audience: those familiar with WAC or those who learn from comparisons
+
+Access control is fundamentally about stating `who` can `do what` to `what resource`.
+
+## ShEx for WAC:
+Web Access Control (WAC) uses a simple schema to make those assertions directly in RDF:
+
+``` shex
+PREFIX acl: <http://www.w3.org/ns/auth/acl#>
+PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+
+<AclShape> {
+  # who
+  (  acl:agentClass @<GroupShape> OR [foaf:Agent] ;
+    | acl:agent IRI
+    | acl:origin IRI
+  )+ ;
+
+  # can do what
+  acl:mode [acl:Read acl:Write acl:Append acl:Control] + ;
```

Comment:
The value set for acl:mode is overly restrictive as per ACL Ontology:

```turtle
    acl:mode     a rdf:Property;
         :comment "A mode of access such as read or write.";
         :domain acl:Authorization;
         :label "access mode";
         :range :Class .
```

The rule may reflect deployments however ACL permits any class eg. `acl:Create`, `acl:Delete`.

Reply:
note: the acl Ontology would be a lot clearer if it restricted the range to the subclasses of  acl:Access
https://github.com/solid/authorization-panel/issues/187